homeworks/week2/Huy_Anh.java

```diff
@@ -0,0 +1,58 @@
+import java.util.*;
+
+class Main {
+    public static void main(String[] args) {
+        System.out.println(unorderedRemoveDuplicate(null)); //null
+        System.out.println(orderedRemoveDuplicate(null)); //null
+        System.out.println(unorderedRemoveDuplicate("")); //blank
+        System.out.println(orderedRemoveDuplicate("")); //blank
+        System.out.println(unorderedRemoveDuplicate("doonneeee")); //deno
+        System.out.println(orderedRemoveDuplicate("doonneeee")); //done
+        System.out.println(unorderedRemoveDuplicate("paragraphs")); //aghprs
+        System.out.println(orderedRemoveDuplicate("paragraphs")); //parghs
+        System.out.println(unorderedRemoveDuplicate("yay!!@#@'")); //!#'@ay
+        System.out.println(orderedRemoveDuplicate("yay!!@#@'")); //ya!@#'
+        System.out.println(unorderedRemoveDuplicate("hello world!")); // !dehlorw
+        System.out.println(orderedRemoveDuplicate("hello world!")); //helo wrd!
+    }
+
+    public static String orderedRemoveDuplicate(String inputString) {
```

Comment:
You changed the problem signature which is not recommended. I could have make the problem more clear.